### DISCUSSION TITLE
macbook pro 2011

### DISCUSSION THREAD

**Discussion by Meteo07** [chris1111/Legacy-Video-patch#31] [Q&A]
Hii i'm under catalina 10.15.7 macbook pro early 2011 with catalina patcher 1.4.4, do i install this fix (legacy video patcher)? 

regards

**Comment by chris1111** [comment]
Why install this fix if you use dosdude1 patcher

**Reply by Meteo07** [reply]
i don't know if dosdude give a full acceleration hardware...

**Comment by chris1111** [comment]
you don't know 🥸

**Reply by Meteo07** [reply]
i don't know...

**Comment by chris1111** [comment]
If you don't know how the rest of the world can know

**Comment by Meteo07** [comment]
i think another one of me has a macbook with dosdude patcher... and maybe, they know...
